railseventstore.org/source/community.html.md

```diff
@@ -11,3 +11,56 @@ Any kind of contribution is welcomed.
 * [Create a new issue](https://help.github.com/articles/creating-an-issue/), assuming one does not already exist.
 * Clearly describe the problem including steps to reproduce when it is a bug.
 * If possible provide a Pull Request with failing test case.
+
+## Prepare a Pull Request
+
+* Fork the [RailsEventStore monorepo](https://github.com/RailsEventStore/rails_event_store)
+
+    ```
+    git clone git@github.com:RailsEventStore/rails_event_store.git
+    cd rails_event_store
+    ```
+
+* Make sure you have all latest changes or rebase your forked repository master branch with RailsEventStore master branch
+
+    ```
+    cd rails_event_store
+    make rebase
+    ```
+
+* Create a pull request branch
+
+    ```
+    git checkout -b new_branch
+    ```
+
+* Implement your feature, don't forget about tests & documentation (to see how to work with documentation files check [documentation's readme ](https://github.com/RailsEventStore/rails_event_store/blob/master/railseventstore.org/README.md)
+
+* Make sure your code pass all tests
+
+    ```
+    make test
+    ```
+
+    You could test each project separatelly, just enter the project folder and run tests (`make test` again) there.
```

Comment:
"separately"

Reply:
Nice findings, isn't it equally fast to pull this branch and fix them?

Reply:
@paneq haha - I thought the same, but saw the email fly by and was in the middle of something else. Sorry!! :-)